spec/rails_spec.rb

```diff
@@ -31,6 +31,7 @@ def run(command, options = {})
           run ["bundle", "exec", "rake", "db:setup", "parallel:create"]
```

Comment:
@grosser Should this line be changed to use `parallel:setup` instead of adding a newline?

Reply:
Nevermind, doing so wouldn't work and would result in a test failure:

```
     Failure/Error: raise "FAILED #{command}\n#{result}" if $?.success? == !!options[:fail]
     
     RuntimeError:
       FAILED ["bundle", "exec", "rake", "parallel:prepare"]
       Run `bin/rails db:migrate` to update your database then try again.
       You have 1 pending migration:
         20220322153446 CreateUsers
     # ./spec/rails_spec.rb:10:in `run'
     # ./spec/rails_spec.rb:34:in `block (5 levels) in <top (required)>'
     # ./spec/rails_spec.rb:23:in `block (4 levels) in <top (required)>'
     # ./spec/rails_spec.rb:22:in `chdir'
     # ./spec/rails_spec.rb:22:in `block (3 levels) in <top (required)>'
     # ./spec/spec_helper.rb:213:in `block (2 levels) in <top (required)>'
```